templates/form/formButtons.tpl

```diff
@@ -30,9 +30,9 @@
 					cancelUrl=$FBV_cancelUrl}
 		{/if}
 		{if $FBV_formReset}
-			{fbvElement type="link" class="resetButton" id="resetButton"|concat:"-":uniqid label=$FBV_cancelText}
+			<a href="#" id="resetButton-{$uniqid}" class="resetButton">{translate key=$FBV_cancelText}</a>
```

Comment:
Hmm, it's verbose, but...

    {assign var=resetButtonId value="resetButton-"|uniqid}
    <a href="#" id="{$resetButtonId}" ...

...should do it.